Sources/Actions/Profile/ShowPermissions.php

```diff
@@ -78,8 +78,8 @@ public function execute(): void
 		// Load all the permission profiles.
 		Permissions::loadPermissionProfiles();
 
-		Board::$info->id = empty(Board::$info->id) ? 0 : (int) Board::$info->id;
-		Utils::$context['board'] = Board::$info->id;
+		$board = empty(Board::$info->id) ? 0 : (int) Board::$info->id;
```

Comment:
This error I got while trying to see a user's permissions.  We can reassign it back like that, so I localized it.  2.1 code reassigned it back into $board,